appeals/web/src/server/lib/mappers/appeal/submappers/ip-comments.mapper.js

```diff
@@ -0,0 +1,28 @@
+import { mapDocumentStatus } from '#lib/display-page-formatter.js';
+
+/** @type {import('../appeal.mapper.js').SubMapper} */
+export const mapIpComments = ({ appealDetails, currentRoute }) => {
+	return {
+		id: 'ip-comments',
+		display: {
+			tableItem: [
+				{
+					text: 'Interested party comments'
+				},
+				{
+					text: mapDocumentStatus(appealDetails?.documentationSummary?.ipComments?.status)
+				},
+				{
+					text: 'Not applicable'
+				},
+				{
+					html: `<a href="${currentRoute}/interested-party-comments" data-cy="review-ip-comments" class="govuk-link">${
+						appealDetails?.documentationSummary?.ipComments?.status === 'received'
+							? 'Review'
+							: 'Add'
+					} <span class="govuk-visually-hidden">I P comments</span></a>`
```

Comment:
Very minor, the convention is for the visually hidden text to replicate the row label text ("Interested party comments" in this case)